packages/ui/src/views/unified-pipeline-studio/components/form-inputs/common/InputLabel.tsx

```diff
@@ -15,7 +15,7 @@ function InputLabel(props: InputLabelProps): JSX.Element | null {
 
   const labelText = required && label ? `${label} *` : label
 
-  return <Label className={cn('mb-2.5 block', className)}>{labelText}</Label>
+  return <Label className={cn('mb-2', className)}>{labelText}</Label>
```

Comment:
Thinking out loud: Can we add `mb-2` as the default margin to the `Label` component? We're adding this in most places where the `Label` is used, and setting it by default could help reduce redundancy. 

cc: @abhinavrastogi-harness